tests/foreman/ui/test_host.py

```diff
@@ -956,6 +958,176 @@ def test_positive_search_by_org(self):
                 )
             )
 
+    @run_only_on('sat')
+    @tier2
+    def test_positive_sort_by_name(self):
+        """Create some Host entities and sort them by name ascendingly and then
+        descendingly
+
+        :id: 12f75ef9-23e6-48be-80ed-b354e8ac212b
+
+        :expectedresults: Host entities are sorted properly
+
+        :CaseImportance: High
+
+        :BZ: 1268085
+
+        :CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        name_list = [gen_string('alpha', 20).lower() for _ in range(5)]
+        for name in name_list:
+            entities.Host(name=name, organization=org).create()
```

Comment:
'Fake' host creation takes very long time due to nature of `Host.create_missing()` - it creates lots of entity dependencies. Since you don't need totally different hosts here and the purpose of test is to verify sorting - please run` create_missing()` once and pass all created entities to all 20 host creation commands, that should greatly improve performance (i'm eager to see the difference in test execution time 🙂 )

Reply:
@abalakh just a correction :) there is 5 host not 20, but this may not change your judgment